src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -108,39 +108,35 @@ public function __construct(
         $this->orderShippingMethodSelectionRequirementChecker = $orderShippingMethodSelectionRequirementChecker;
         $this->orderPaymentMethodSelectionRequirementChecker = $orderPaymentMethodSelectionRequirementChecker;
 
+        $this->orderExampleFactory = $orderExampleFactory;
+        if ($orderExampleFactory == null) {
+            $this->orderExampleFactory = new OrderExampleFactory(
```

Comment:
Can we extract this object creation to the private method? It would be much more readable